app/src/main/res/layout-land/ongoing_story_card.xml

```diff
@@ -59,8 +63,9 @@
           android:layout_marginTop="8dp"
           android:layout_marginEnd="8dp"
           android:ellipsize="end"
-          android:maxLines="1"
+          android:lines="2"
           android:text="@{viewModel.ongoingStory.storyName}"
+          android:textStyle="bold"
```

Comment:
Also for this change we need mock link

Reply:
Text is bold in the mock link in the link https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0e6ba726-1930-4f39-9c73-51df10a1c766/specs/ .

Reply:
Thanks.